client/js/options.js

```diff
@@ -22,156 +35,263 @@ const options = {
 	notifyAllMessages: false,
 	showSeconds: false,
 	statusMessages: "condensed",
-	theme: $("#theme").attr("href").replace(/^themes\/(.*).css$/, "$1"), // Extracts default theme name, set on the server configuration
+	theme: $theme.attr("href").replace(/^themes\/(.*).css$/, "$1"), // Extracts default theme name, set on the server configuration
 	media: true,
-	userStyles: userStyles.text(),
+	userStyles: "",
 };
-let userOptions = JSON.parse(storage.get("settings")) || {};
 
-for (const key in options) {
-	if (userOptions[key] !== undefined) {
-		options[key] = userOptions[key];
+const noSync = ["syncSettings"];
+// Reserved for things like "highlights".
```

Comment:
Nitpick: it's not clear which this is referring to. I initially thought it was noSync it was talking about.

Reply:
The comment you mean?

Reply:
Yeah